### DISCUSSION TITLE
Scrolling in the app too slow?

### DISCUSSION THREAD

**Discussion by Domjan1911** [FreeTubeApp/FreeTube#3854] [Q&A] [answered]
When I use a scroll wheel on other programs/linux it feels atleast double as fast as on the Freetube app, and I'm wondering if there's a fix for it.

**Comment by absidue** [comment, marked as answer]
According to a quick internet search, scrolling has always been slower in Chromium based browsers on Linux than on Windows. As FreeTube uses Electron it presumably suffers from the same issue.

This AskUbuntu answer recommends using `imwheel` to fix it: https://askubuntu.com/a/991680
